azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/handlers/artifact/FunctionArtifactHelper.java

```diff
@@ -40,6 +43,15 @@ public static File createFunctionArtifact(final String stagingDirectoryPath) thr
         return zipPackage;
     }
 
+    public static void updateAppSetting(final DeployTarget target, final String key, final String value) throws MojoExecutionException {
+        final WebAppBase webAppBase = target.getApp();
+        if (!(webAppBase instanceof FunctionApp)) {
```

Comment:
why webApp instance of Function the name is not proper

Reply:
Renamed